### PR TITLE
Yelp fred

### PR DESCRIPTION
**Pull request by IamTheCry**
Fred is now able to route food requests. Client still needs component to render yelp data.

### CODE REVIEW

File: server/apis/restaurant.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const request = require('request-promise');
+const fredTools = require('../fred/parseTools');
+require('dotenv').config();
+
+
+const restaurant = {
+
+  getResponse: (loc, placeStr, originalStr) => {
+    var options = {
```
Comment (scottyoon221) [inline server/apis/restaurant.js]: change var to let

File: server/apis/weather.js [hunk 2]
```diff
@@ -10,8 +10,7 @@ const weather = {
     console.log('ran weather');
 
     if(!placeStr) {
-      let coordinates = JSON.parse(loc);
-      var query = 'http://api.openweathermap.org/data/2.5/weather?lat=' + coordinates.lat + '&lon=' + coordinates.lon;
+      var query = 'http://api.openweathermap.org/data/2.5/weather?lat=' + loc.lat + '&lon=' + loc.lon;
```
Comment (scottyoon221) [inline server/apis/weather.js]: change var to let

Comment (scottyoon221) [inline server/apis/weather.js]: also declare query outside of if statement

File: server/apis/restaurant.js (hunk 1 above)
Comment (IamTheCry) [inline server/apis/restaurant.js]: Done

File: server/apis/weather.js (hunk 2 above)
Comment (IamTheCry) [inline server/apis/weather.js]: Can't, block scope messes up query declaration.

**Pull request merged**